### PR TITLE
GLOB-79796: Migrate to CodeArtifact

### PR DESCRIPTION
**Pull request by d-globality**
- Node 20
- Migrate from JFrog to CodeArtifact

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -6,31 +6,30 @@ name: default
...
-      ARTIFACTORY_USERNAME:
-        from_secret: ARTIFACTORY_USERNAME
-      ARTIFACTORY_PASSWORD:
-        from_secret: ARTIFACTORY_PASSWORD
+      CODEARTIFACT_AUTH_TOKEN:
+        from_secret: CODEARTIFACT_AUTH_TOKEN
     commands:
-        - curl -u$ARTIFACTORY_USERNAME:$ARTIFACTORY_PASSWORD https://globality.jfrog.io/globality/api/npm/auth > .npmrc
-        - echo "registry=https://globality.jfrog.io/globality/api/npm/npm" >> .npmrc
+      - echo "registry=https://globality-308798440167.d.codeartifact.us-east-1.amazonaws.com/npm/globality-npm-local/" > .npmrc
```
Comment (tor-globality) [inline .drone.yml:22]: I'm thinking we should constrain CodeArtifact usage to `@globality` scope?

File: yarn.lock [hunk 2]
```diff
@@ -1138,15 +1138,15 @@ atob@^2.1.1:
 
 available-typed-arrays@^1.0.7:
   version "1.0.7"
-  resolved "https://registry.yarnpkg.com/available-typed-arrays/-/available-typed-arrays-1.0.7.tgz#a5cc375d6a03c2efc87a553f3e0b1522def14846"
+  resolved "https://globality-308798440167.d.codeartifact.us-east-1.amazonaws.com/npm/globality-npm-local/available-typed-arrays/-/available-typed-arrays-1.0.7.tgz#a5cc375d6a03c2efc87a553f3e0b1522def14846"
```
Comment (tor-globality) [inline yarn.lock:1141]: Public packages should probably just come from Yarn registry, like always

**Comment by tzur-glob** [comment]
Per this [slack thread](https://globalityinc.slack.com/archives/CEZDTHYNL/p1727111500151279), we should move the nodule-* repos to a single repo, and not publish the public repos to Code Artifact

**Comment by tzur-glob** [comment]
Will be covered by https://globality.atlassian.net/browse/GLOB-79364

**Pull request closed without merging**